hc/api/tests/test_check_model.py

```diff
@@ -5,29 +5,30 @@
 from hc.api.models import Check
 
 
+check = Check()
```

Comment:
Instead of having this here, line[8] you could have placed it in ```def setUp``` method.

Reply:
yes, point noted.